packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/applyChange.ts

```diff
@@ -75,5 +75,8 @@ export default function applyChange(
     if (wasResizedOrCropped || state == ImageEditInfoState.FullyChanged) {
         image.width = targetWidth;
         image.height = targetHeight;
+        // Remove width/height style so that it won't affect the image size, since style width/height has higher priority
+        image.style.removeProperty('width');
```

Comment:
Hmm, should we keep the CSS style and HTML attribute have the same value

Reply:
We remove this properties due this issue https://github.com/microsoft/roosterjs/pull/1899

Reply:
I see.

Do we also need to change Content Model for image? Because we will write back image size using CSS

packages-content-model\roosterjs-content-model-dom\lib\formatHandlers\common\sizeFormatHandler.ts

Reply:
It seems content model will write back the image after when quit editing, so the style width/height will be already removed.